_component_design/tooltip.md

```diff
@@ -13,28 +13,292 @@ last-modified: 2017-08-17
 
--   The tooltip disappears when mouse hovers away, or when clicked in the icon.
-    Tooltip does not reactivate on click.
+- **Icon:** Use this trigger variation if a recognizable icon on the application informs a user that additional information is available for consumption if needed. For example, A ‘?’ for help or ‘$’ for pricing.
+- **Table text:** This variation is used to reveal additional information within a table. By hiding information in tooltips in a table, the tables are easily consumable and only display additional information if required. 
+	- **Note:** May apply to both plain text cells and link text cells.
 
--   Tooltip should only be used on icon images, text, and table rows; use when
-    see fit.
+- **Data visualization:** Use this trigger variation to present robust data sets aggregated into a data visualization.
 
--   Max width for the tooltip is 440px.
 
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Examples of tooltips generated from various trigger types."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-triggers.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Text conventions
+{: .hxSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Tooltip"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Tooltips/tooltip.svg){:width="100%"}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{: .hxSubSectionTitle}
+
+- To keep the tooltip text concise, you should use short sentences and sentence fragments. 
+- Try to keep your tooltips under three lines of text including a link if you need one.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Most tooltips should be very short and only provide enough information for the user to better understand the context. Tooltips can also be used to reveal information that is truncated or hidden by default."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-text-conventions-1.svg){:width="100%"}
+
 {% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tooltip specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Tooltips/tooltip-specs.svg){:width="100%"}
+</div>
+
+## Tooltip variations
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Help text with a link
+{: .hxSubSectionTitle}
+
+Use this variation when a tooltip provides access to a singular external webpage.
+
+- This link should always be an external link and should open in a new tab. 
+- Be sure you always include an external link icon.
+- If you need additional inputs or interactions on a tooltip, for example, a dropdown or selector, use the larger popover pattern.
```

Comment:
s/`dropdown`/`drop-down`

Also, link the popover doc.

Reply:
updated